velox/functions/prestosql/window/tests/NtileTest.cpp

```diff
@@ -84,7 +85,7 @@ TEST_F(NtileTest, multiInput) {
 
 TEST_F(NtileTest, multiInputWithSortOrders) {
   NtileTest::testWindowFunction(
-      {makeSinglePartitionVector(75), makeSinglePartitionVector(50)},
+      {makeSinglePartitionVector(50), makeSinglePartitionVector(50)},
```

Comment:
This change is not needed after https://github.com/facebookincubator/velox/pull/3916

Reply:
Reverted, apologies for missing this after rebase.